### PR TITLE
Optional string in fillna

### PR DESCRIPTION
**Pull request by Andre-Sacilotti**
Hey, i added an optional argument in fillna, for those, like me, who need to fill missing values with an custom string.

Example:

```python
>>> hero.fillna(s, "Custom String!!")
0                I'm
1    Custom String!!
2    Custom String!!
3             You're
dtype: object
```

### CODE REVIEW

**Comment by Andre-Sacilotti** [comment]
Closes #182

File: texthero/preprocessing.py [hunk 1]
```diff
@@ -24,32 +24,39 @@
 
 
 @InputSeries(TextSeries)
-def fillna(s: TextSeries) -> TextSeries:
+def fillna(s: TextSeries, replace_string: Optional[str] = "") -> TextSeries:
```
Comment (mk2510) [inline texthero/preprocessing.py:27]: I wouldn't do the optional type here, either the "" or the typing with Optional[str] but not both, as we don't gain any informations from that.

File: texthero/preprocessing.py [hunk 2]
```diff
@@ -24,32 +24,39 @@
...
     """
-    Replaces not assigned values with empty string.
+    Replaces not assigned values with empty (Default) or given string.
```
Comment (mk2510) [inline texthero/preprocessing.py:29]: the default here as well is too much in my opinion. The user will probably recognise, that "" is an empty string

**Review by mk2510 — COMMENTED** [review]
Looks great overall 🥳 
I just had two small comments for the signature of the function and the docstring. But I like this addition!

File: texthero/preprocessing.py (hunk 2 above)
Comment (Andre-Sacilotti) [reply texthero/preprocessing.py:29]: Hey, thanks for the comments, this help us learn more. I will commit the changes

**Comment by henrifroese** [comment]
[Here](https://github.com/jbesomi/texthero/pull/190/files) we can see that your changes change a lot of "invisible" stuff (whitespace) in the whole preprocessing file, which of course we cannot accept as this leads to doctests failing. Not sure which Black version you are using? You can see [here](https://github.com/jbesomi/texthero/issues/171) that we sadly have problems with the newest version; that might be the issue 😿

**Comment by Andre-Sacilotti** [comment]
> [Here](https://github.com/jbesomi/texthero/pull/190/files) we can see that your changes change a lot of "invisible" stuff (whitespace) in the whole preprocessing file, which of course we cannot accept as this leads to doctests failing. Not sure which Black version you are using? You can see [here](https://github.com/jbesomi/texthero/issues/171) that we sadly have problems with the newest version; that might be the issue crying_cat_face

oh, i just saw the travis ci build failed, i'm using black-19.10b0, i'm trying to resolve this

**Comment by Andre-Sacilotti** [comment]
Okay, i think its working right now. Any tips or comments will be very helpful.

**Comment by jbesomi** [comment]
Hey @Andre-Sacilotti, looks great! Thank you!

And welcome to Texthero as a contributor, you can open a new PR and add your name in the README.md file! 👍

**Pull request merged**